trending_deploy/deploy.py

```diff
@@ -151,6 +151,7 @@ def undeploy_model(model_name: str) -> bool:
         for endpoint in endpoints:
             if endpoint.repository == model_name and endpoint.name.startswith(ENDPOINT_PREFIX):
                 endpoint.delete()
+                delete_collection_item(COLLECTION_SLUG, item_id=model_name)
```

Comment:
Sadly it looks like this only accepts `item_object_id`, i.e. the ID fo the item in the collection, not the id of the item on the Hub (like repo_id or paper_id): https://huggingface.co/docs/huggingface_hub/main/en/package_reference/hf_api#huggingface_hub.HfApi.delete_collection_item

I think you have to end up using https://huggingface.co/docs/huggingface_hub/main/en/package_reference/hf_api#huggingface_hub.HfApi.get_collection and then searching for the collection item with the matching model_name, sadly :/

Reply:
broooooooooooooo